components/multimedia/gst-plugins-good1/manifests/sample-manifest.p5m

```diff
@@ -136,6 +139,7 @@ file path=usr/lib/gstreamer-1.0/libgstsmpte.so
 file path=usr/lib/gstreamer-1.0/libgstsoup.so
 file path=usr/lib/gstreamer-1.0/libgstspectrum.so
 file path=usr/lib/gstreamer-1.0/libgstspeex.so
+file path=usr/lib/gstreamer-1.0/libgsttaglib.so
 file path=usr/lib/gstreamer-1.0/libgsttwolame.so
 file path=usr/lib/gstreamer-1.0/libgstudp.so
 file path=usr/lib/gstreamer-1.0/libgstvideo4linux2.so
```

Comment:
When I run gmake sample-manifest it will have two additional libs:
file path=usr/lib/gstreamer-1.0/libgstwavpack.so
file path=usr/lib/$(MACH64)/gstreamer-1.0/libgstwavpack.so